ambuda/checks.py

```diff
@@ -0,0 +1,76 @@
+"""Sanity checks that run on application startup."""
+
+
+import sys
+
+from click import style
+from sqlalchemy import create_engine
+from sqlalchemy import inspect
+
+from ambuda.models.base import Base
+
+
+def _warn(text: str = ""):
+    """Print a warning message to the terminal."""
+    print(style(text, fg="red"))
+
+
+def check_app_schema_matches_db_schema(database_uri: str):
+    """Check that our application tables and database tables match.
+
+    Currently, we apply the following checks:
+    - All app tables exist in the database.
+    - A given app table and its corresponding database table have the same
+      column names.
+
```

Comment:
Maybe explicitly note that this won't catch all inconsistencies between the database and application tables? For example, it won't catch a migration that just altered an existing column